source/javascripts/components/StepItem/AddStepItem.scss

```
@@ -57,6 +57,12 @@
 					font-weight: 400;
 					color: $def-mediumdarkgrey;
 
+					p {
+						overflow: hidden;
+						white-space: nowrap;
+						text-overflow: ellipsis;
```

I have a mixin for that ™

```suggestion
						@include ellipsis-oneliner;
```